sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/models/IncludedPathTest.java

```diff
@@ -1,40 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-package com.azure.cosmos.models;
-
-import org.assertj.core.api.Assertions;
-import org.testng.annotations.Test;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class IncludedPathTest {
-
-    @Test(groups = {"unit"})
-    public void deserialize() {
```

Comment:
now that custructor for parsing json is removed, could you please add a test which sets different fields using direct api invocation?

Reply:
added